src/components/__test__/Missions.test.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] Please remove the ESLint disabler from your file. If you are having difficulty resolving the issue, kindly provide a description of the error you are experiencing.